Frontend/src/components/Table/Cell.svelte

```diff
@@ -1,7 +1,7 @@
 <script>
   export let backgroundColor = "white";
   export let isImage = false;
-  export let value = "";
+  export let valueFunction = async () => "";
```

Comment:
Why does this have to be async?

Reply:
Because it‘s awaited on line 36. This is necessary so that a valueFunction loading the number of active rentals from the database can be passed to a cell. This needs to happen asynchronously in a cell, so that values can be lazy loaded.